src/runtime-tools/linux/onefuzz.service

```diff
@@ -1,12 +1,14 @@
 # onefuzz systemd startup script
 [Unit]
 Description=onefuzz
+StartLimitIntervalSec=20
```

Comment:
This would be great if it only throttled within the interval, and had no effect on retry policy after a cool-off period. But to echo @bmc-msft, I think we should remove these settings, since:

> Units which are started more than `burst` times within an interval time span are **not permitted to start any more.**

> Note that units which are configured for `Restart=`, and which reach the start limit are **not attempted to be restarted anymore** [...]

([source](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#StartLimitIntervalSec=interval))

Fortunately, with an increased `RestartSec`, I think this is probably redundant anyway.